stringify/stringify.go

```diff
@@ -0,0 +1,22 @@
+package stringify
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+func Data(data any) string {
```

Comment:
A comment would be nice. Also personally I find `stringify.Any(...)` to be a better name than `stringify.Data(...)` but it's totally subjective I guess :slightly_smiling_face:

Reply:
Thanks for pointing out. Added the comment. Also, `Any` is a definitely better name.